ci/infra/vmware/variables.tf

```diff
@@ -119,3 +119,8 @@ variable "cpi_enable" {
   default     = false
   description = "Enable CPI integration with vSphere"
 }
+
+variable "hostname_from_dhcp" {
+  default     = false
```

Comment:
Do we want to change the default behavior ? if so we should not forget to add this in the Release note.

Reply:
Good catch!
No, we don't want to change the default behavior, revert the value.